Master RMarkdown Document & Render Code/Locality_Profiles_Master_Markdown.Rmd

```diff
@@ -203,13 +203,12 @@ x <- x + 1
 
 \newline
 
-```{r echo = FALSE, fig.cap= 'A map of the health and social care partnership including a breakdown on localities with the locations of all the GP practices, care homes, emergency depertments and minor injury units.'}
-knitr::include_graphics(map, rel_path = FALSE)
+```{r echo = FALSE, fig.width=6, fig.cap= "A map of the health and social care partnership including a breakdown on localities with the locations of all the GP practices, care homes, emergency depertments and minor injury units."}
+service_map
 
 x <- x + 1
 ```
 
-###### ED = Emergency Department, MIU = Minor Injuries Unit (or other)
```

Comment:
This line is being removed in the 'main' Rmd, assumedly because it's not included in the map? However, it's not been deleted from the 'testing' Rmd. Can you make sure both Rmd docs are the same for the bits around the map please?